### DISCUSSION TITLE
Add subtitles to ogg files

### DISCUSSION THREAD

**Discussion by Franziskus** [JonathanFly/bark#80] [Ideas]
First I want to say I love this tool it does an amazingly job.
I was thinking since we already have the text, I wonder if it would be possible to add them as subtitles to the ogg?
Not matching word for word, but machining the snipped generated by bark.

I was also thinking about a chapters too. but if we have subtitles with timestamps it would be even possible to generate them based on the subtitles itself.

**Comment by JonathanFly** [comment]
Sure, let me know how any specific audio book needs. Not something I have generated so I don't know what's missing.

**Reply by Franziskus** [reply]
I was thinking about it maybe you could simply generate some subtitle file like an *.str. And the rest we can do in other tools. It would simply help if we have a timestamp for the generated texts.

**Comment by JonathanFly** [comment]
> I was thinking about it maybe you could simply generate some subtitle file like an *.str. And the rest we can do in other tools. It would simply help if we have a timestamp for the generated texts.

I could generate accurate subtitles with Whisper, or I could stick your text prompt in the subtitles, which would roughly accurate to the 10 second audio chunk. Or both I guess.

**Reply by Franziskus** [reply]
I agree, that whisper is an option. But in my experience, bark is not perfect sometimes and produces some random output. This would be one of the reasons to be able to have a feedback of timestamp and text.
